packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/components/FlipbookControls.js

```diff
@@ -0,0 +1,151 @@
+import { Button, Box, Grid } from 'grommet'
+import { FormNext, FormPrevious } from 'grommet-icons'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled, { withTheme } from 'styled-components'
+import { useTranslation } from 'react-i18next'
+
+import locationValidator from '../../../helpers/locationValidator'
+
+const StyledButton = styled(Button)`
+  & > div {
+    flex-direction: column;
+  }
+`
+
+const backgrounds = { dark: 'dark-3', light: 'neutral-6' }
+
+const FlipbookControls = ({
+  currentFrame = 0,
+  locations = [],
+  onFrameChange = () => true,
+  theme
+}) => {
+  const { t } = useTranslation('components')
+
+  const handleKeyDown = (event) => {
+    switch (event.key) {
+      case 'ArrowRight': {
+        event.preventDefault()
+        event.stopPropagation()
+        handleNext()
+        break
+      }
+      case 'ArrowLeft': {
+        event.preventDefault()
+        event.stopPropagation()
+        handlePrevious()
+        break
+      }
+      default: {
+        return true
+      }
+    }
+  }
+
+  const handleFrameChange = frameIndex => {
+    onFrameChange(frameIndex)
+  }
+
+  const handlePrevious = () => {
+    if (currentFrame > 0) {
+      handleFrameChange(currentFrame - 1)
+    }
+  }
+
+  const handleNext = () => {
+    if (currentFrame < locations.length - 1) {
+      handleFrameChange(currentFrame + 1)
+    }
+  }
+
+  return (
+    <Box background={backgrounds}>
+      <Grid columns={['120px', 'flex']} pad='10px'>
+        {/** Play/Pause & Speed go here */}
+        <Box />
+
+        {/** Image Thumbnails */}
+        <Box direction='row' justify='between'>
+          <StyledButton
+            disabled={currentFrame === 0}
+            icon={<FormPrevious />}
+            label={t(
+              'SubjectViewer.MultiFrameViewer.FrameCarousel.previousFrameLabel'
+            )}
+            onClick={() => handlePrevious()}
+            style={{
+              border: 'none',
+              padding: 0
+            }}
+          />
+          <Box
+            aria-label='Image thumbnails'
+            direction='row'
+            gap='5px'
+            pad='5px'
+            role='tablist'
+          >
```

Comment:
Should there be a visual style change when tablist is focused? I tried it briefly using `onFocus` and `onBlur` with the tab buttons, but wasn't super happy with the UX. We might need a design suggestion.

Reply:
There’s a visual style already built into the theme. Focused elements have a teal outline. You can see it on the navigation links when you tab through the top of the page.

Reply:
The tablist itself won’t be focused (it doesn’t have a tabindex) but the individual buttons should have a focus indicator. 

You can see that on the Task and Tutorial tabs when you tab through the classifier, which have the default focus ring for buttons.

Reply:
Sounds good! The tab buttons do have a focus indicator